src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -261,20 +274,42 @@ private static TextSpan GetCompletionItemSpan(SourceText text, int position)
             }
         }
 
+        private bool TryCreateSpecialTypeItem(
```

Comment:
Thanks @dpoeschl. I added some comments and did a rename--let me know if that makes things more clear.